### PR TITLE
Initial API Endpoint Integration

### PR DESCRIPTION
**Pull request by sei-aderr**
Addresses #184, partially addresses #185, #186. Able to read and display versions + models but not add new ones through the UI

### CODE REVIEW

File: mlte/frontend/nuxt-app/pages/index.vue [hunk 1]
```diff
@@ -158,25 +163,21 @@ const path = ref([
   },
 ]);
 
-const namespaces = ref(["Default", "TEST 1", "Super longg purposes"]);
-
-const selectedNamespace = ref("Default");
+const { data: namespaceOptions } = await useFetch<string[]>(
+  "http://localhost:8080/api/namespace",
```
Comment (turingcompl33t) [inline mlte/frontend/nuxt-app/pages/index.vue:167]: Not necessary to update this for this PR, but we will need to make this configurable to alllow users to run the store in locations other than `localhost:8080`. We can default to some value like this in development mode, but when run from the CLI, we should accept an argument that is the address of the artifact store.

**Review by turingcompl33t — APPROVED** [review]
Nice work Alex. Our first glimmers of an end-to-end MLTE system!

File: mlte/frontend/nuxt-app/pages/index.vue (hunk 1 above)
Comment (turingcompl33t) [reply mlte/frontend/nuxt-app/pages/index.vue:167]: I created an issue for this.

Comment (sei-aderr) [reply mlte/frontend/nuxt-app/pages/index.vue:167]: Yes, still want to get that implemented. I am not sure that there is going to be a simple way to make that happen so I just wanted to get this along with the rest of the frontend implemented before going down that rabbit hole.

**Pull request merged**